<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WallDisplay</class>
 <widget class="QMainWindow" name="WallDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>442</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bitstream Charter</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>WallDisplay</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWebView" name="webView">
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
       </font>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_settings"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionS_top_interval"/>
    <addaction name="actionStart_interval"/>
    <addaction name="separator"/>
    <addaction name="actionHide_menu"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Info"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_settings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionS_top_interval">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop interval</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionStart_interval">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start interval</string>
   </property>
  </action>
  <action name="actionHide_menu">
   <property name="text">
    <string>Hide menu</string>
   </property>
  </action>
  <action name="action_Info">
   <property name="text">
    <string>&amp;Info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>WallDisplay</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHide_menu</sender>
   <signal>triggered()</signal>
   <receiver>menuBar</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
